multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -159,45 +159,25 @@ coreos:
         ExecStartPre=/usr/bin/curl http://127.0.0.1:8080/version
         ExecStart=/opt/bin/install-kube-system
 
-{{ if .UseCalico }}
-    - name: install-calico-system.service
-      command: start
-      content: |
-        [Unit]
-        Requires=kubelet.service docker.service
-        After=kubelet.service docker.service
-
-        [Service]
-        Type=simple
-        StartLimitInterval=0
-        Restart=on-failure
-        ExecStartPre=/usr/bin/curl http://127.0.0.1:8080/version
-        ExecStart=/opt/bin/install-calico-system
-{{ end }}
-
 write_files:
+  - path: /opt/bin/kubectl
+    permissions: 0755
+    content: |
+      # source this file to set up a kubectl alias
+      sudo rkt run \
+        --net=host \
+        --volume bk,kind=host,source=/srv/kubernetes \
+        --mount volume=bk,target=/srv/kubernetes \
+        --insecure-options=image docker://{{.HyperkubeImageRepo}}:{{.K8sVer}} \
+        --exec /hyperkube -- kubectl $@
+
   - path: /opt/bin/install-kube-system
     permissions: 0700
     owner: root:root
     content: |
       #!/bin/bash -e
-      /usr/bin/curl  -H "Content-Type: application/json" -XPOST \
-      -d @"/srv/kubernetes/manifests/kube-dns-rc.json" \
-      "http://127.0.0.1:8080/api/v1/namespaces/kube-system/replicationcontrollers"
-
-      /usr/bin/curl  -H "Content-Type: application/json" -XPOST \
-      -d @"/srv/kubernetes/manifests/kube-dashboard-rc.json" \
-      "http://127.0.0.1:8080/api/v1/namespaces/kube-system/replicationcontrollers"
-
-      /usr/bin/curl  -H "Content-Type: application/json" -XPOST \
-      -d @"/srv/kubernetes/manifests/heapster-de.json" \
-      "http://127.0.0.1:8080/apis/extensions/v1beta1/namespaces/kube-system/deployments"
-
-      for manifest in {kube-dns,heapster,kube-dashboard}-svc.json;do
-          /usr/bin/curl  -H "Content-Type: application/json" -XPOST \
-          -d @"/srv/kubernetes/manifests/$manifest" \
-          "http://127.0.0.1:8080/api/v1/namespaces/kube-system/services"
-      done
+
+      /opt/bin/kubectl apply -f /srv/kubernetes/manifests
```

Comment:
I think we should be using `create` not `apply` otherwise an admin can never update their addons (this will be run on every boot). `create`, I believe, will only create if nothing exists - if addon is already present it is a no-op.